tests/test_base.py

```diff
@@ -0,0 +1,311 @@
+# copy AS-IS from django repository at stable/3.2.x
+# https://github.com/django/django/blob/754af45/tests/migrations/test_base.py
```

Comment:
copy AS-IS from django repository at stable/3.2.x
https://github.com/django/django/blob/754af45/tests/migrations/test_base.py